sync-core/src/main/java/com/cloudant/sync/query/IndexManager.java

```diff
@@ -94,18 +93,32 @@ public IndexManager(Datastore datastore) {
         queue = Executors.newSingleThreadExecutor();
 
         final String filename = datastore.extensionDataFolder(EXTENSION_NAME) + File.separator
-                                                                        + "indexes.sqlite";
+                                                                              + "indexes.sqlite";
         SQLDatabase sqlDatabase = null;
         try {
             sqlDatabase = queue.submit(new Callable<SQLDatabase>() {
```

Comment:
Not for this PR, but should be done while refactoring this class. It shouldn't hold on to sqlDatabase, instead it should be using an SQLDatabaseQueue to interact with the database, @alfinkel do you have plans to refactor parts of this class or do we need another ticket to handle this?